src/Config/Cleanup.targets

```diff
@@ -8,6 +8,7 @@
       <FilesToDelete Include="$(OutputPath)nodes\SharpDX*"/>
       <FilesToDelete Include="$(OutputPath)nodes\HelixToolkit*"/>
       <FilesToDelete Include="$(OutputPath)nodes\Cyotek*"/>
+      <FilesToDelete Include="$(OutputPath)nodes\ProtoGeometry*"/>
```

Comment:
👍

Reply:
so one thing I forgot to mention about this - is that these are not foolproof - these targets are only imported into a few projects, so if another project that builds after copies the specified binaries *again* we'll have the same trouble - we really should have a way to add a target for post build to the entire .sln - but I think thats not straight forward.

Reply:
alternatively, we could import them into all projects as part of our standard import I guess.